CHANGELOG.md

```diff
@@ -4,19 +4,47 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.0.0-beta5 (2020-06-08)
+
+This release adds support for basic LIMIT pushdown, compression on
+distributed hypertables, table access methods on distributed hypertables,
+storage options on distributed hypertables, allow SERIAL columns in
+distributed hypertables and adds support for altering replication factor.
+This release only supports PG11 and PG12.
```

Comment:
A bit repetitive w.r.t. "distribute hypertables". I propose streamlining with some other tweaks.

```suggestion
This release adds new functionality on distributed hypertables, including (but not limited to) basic LIMIT pushdown, manual chunk compression, table access methods
storage options,  SERIAL columns, and altering of the replication factor.
This release only supports PG11 and PG12. Thus, PG9.6 and PG10 are no longer supported.
```

Reply:
Should we add something about API-breaking changes? E.g.,:

`
Note that the 2.0 major release will introduce breaking changes to user functions and APIs. In particular, this beta removes the `cascade` parameter from `drop_chunks` and changes the names of certain GUC parameters. Expect additional breaking changes to be introduced up until the 2.0 release. `